Configurations/00-base-templates.conf

```diff
@@ -92,18 +89,14 @@ my %targets=(
         lib_defines      =>
             sub {
                 my @defs = ();
-                unless ($disabled{"zlib-dynamic"}) {
-                    my $zlib = $withargs{zlib_lib} // "ZLIB1";
-                    push @defs, 'LIBZ=' . (quotify("perl", $zlib))[0];
-                }
+                my $zlib = $withargs{zlib_lib} // "ZLIB1";
+                push @defs, 'LIBZ=' . (quotify("perl", $zlib))[0];
```

Comment:
Er, you can actually drop these lines entirely.  With `zlib-dynamic` removed, the `LIBZ` macro isn't relevant any more.
As a matter of fact, setting the `lib_defines` attribute can be removed complete here.

Reply:
remove the setting of `lib_defines` thanks!